packages/documentation/src/stories/components/forms/validation/validation.stories.ts

```diff
@@ -123,6 +122,7 @@ export const Input: Story = {
         placeholder="Placeholder"
       />
       <label class="form-label" for="Input_1${props.validationState}${props.scheme}">Label</label>
+      <span class="validation-icon"></span>
```

Comment:
Changing the structure of the HTML for HTML/CSS components is a major breaking change because every project has to update every input field in their whole project for the changes to take effect. It would be great to have a solution that does not alter the markup required.

Since input fields cannot have pseudo elements, I was thinking that we maybe can put a pseudo element on the wrapper element around each input field and use `:has` to determine the validity state:

```css
.form-input:has(input[type="text"].is-invalid)::before { content ''; background-image: /*invalid icon*/}
```